src/chain/config.ts

```diff
@@ -33,6 +33,18 @@ const litentryNetwork: ChainConfig = {
         }
       };
     }
+  }, {
+    pattern: "tips.newTip",
+    getPushData() {
+      return {
+        topic: "tips.newTip",
+        data: { deeplink: "litentry://api/litentry_test/tips" },
```

Comment:
All these events will have the same title and body for all networks. This payload should be constructed in a way that can be reused for all networks, the network should come from the env variable (if we decide to keep that approach). Later we can come up with a better format for the links

Reply:
I have changed this on 188e76c please check if it's better now.